PartsAppTests/ExcelHelperTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace PartsAppTests
+{
+    [TestClass]
+    public class ExcelHelperTests
+    {
+        [TestMethod]
```

Comment:
Why are these 3 different methods?

Reply:
replaced by methods:
1) testing input that can be validated
2) testing null, empty string and string with only forbidden characters